frontend/src/APIClients/queries/CourseQueries.ts

```diff
@@ -0,0 +1,30 @@
+import { gql } from "@apollo/client";
+
+export const COURSES = gql`
+  query Courses {
+    courses {
+      id
+      title
+      description
+      image
+      previewImage
+      modules
```

Comment:
![image](https://user-images.githubusercontent.com/41529410/160041523-de983886-b010-4654-b6a6-daa2e1faefdc.png)

I got this error when local hosting your branch. Please add the following to all the `module` attributes (including those in the mutations file)

```
    modules {
        id
        title
        description
        image
        previewImage
        published
        lessons
    }
```

Reply:
fixed!